### PR TITLE
(DOCSP-14576): [iOS] Add UUID to Supported Property Types

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-14576

### Staged Changes (Requires MongoDB Corp SSO)

- [Supported Property Types - iOS SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14576/sdk/ios/data-types/supported-property-types/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/ios/data-types/supported-property-types.txt [hunk 1]
```diff
@@ -130,3 +136,10 @@ Supported Property Types - iOS SDK
       ``CGFloat`` properties are discouraged, as the type is not
       platform independent.
 
+Unique Identifiers
+------------------
+
+ObjectId is a MongoDB-specific 12-bit unique value. UUID is a 16-bit
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/data-types/supported-property-types.txt:142]: I double-checked when I wrote the Android side of this, and it looks like they're actually 12-bytes and 16-bytes, not bits. Big difference!

File: source/sdk/ios/data-types/supported-property-types.txt [hunk 2]
```diff
@@ -130,3 +136,10 @@ Supported Property Types - iOS SDK
       ``CGFloat`` properties are discouraged, as the type is not
       platform independent.
 
+Unique Identifiers
+------------------
+
+ObjectId is a MongoDB-specific 12-bit unique value. UUID is a 16-bit 
+globally-unique value. Both types are :ref:`indexable <ios-index-a-property>` 
+and can be used as :ref:`primary keys <ios-specify-a-primary-key>`.
```
Comment (ccho-mongodb) [inline source/sdk/ios/data-types/supported-property-types.txt:144]: I think "can be used" is passive voice. You can make it active voice by specifying who is taking the action, e.g.:

```suggestion
which you can use as :ref:`primary keys <ios-specify-a-primary-key>`.
```

File: source/sdk/ios/data-types/supported-property-types.txt (hunk 1 above)
Comment (ccho-mongodb) [inline source/sdk/ios/data-types/supported-property-types.txt:142]:
```suggestion
ObjectId is a MongoDB-specific 12-byte unique value. UUID is a 16-byte 
```

**Review by ccho-mongodb — APPROVED** [review]
LGTM, just a couple small fixes suggested.

File: source/sdk/ios/data-types/supported-property-types.txt (hunk 1 above)
Comment (dacharyc) [reply source/sdk/ios/data-types/supported-property-types.txt]: Good note, thank you! Updated.

File: source/sdk/ios/data-types/supported-property-types.txt (hunk 2 above)
Comment (dacharyc) [reply source/sdk/ios/data-types/supported-property-types.txt]: Good point, Chris - reworked this to remove the passive voice. Thanks for spotting that!

**Pull request merged**